app/views/mail/newUser.scala.html

```diff
@@ -6,14 +6,12 @@
 
   @if(!enableAutoVerify) {
     <p>
-      Welcome to your new webKnossos account. You are almost ready to start exploring and to collaboratively
-      annotate large 3D datasets.
+      Welcome to your new webKnossos account. You are almost ready to share and annotate your large 3D datasets.
     </p>
     <p>Your account is currently under review. An administrator will activate you soon.</p>
```

Comment:
Btw: Do we send an email when a user was activated later? Might be unimportant, since the new invite-by-email-feature uses autoEnable, as far as I know, but in general it would make sense..

Reply:
I don't think there is an activation-notifier. Would be helpful, tho. cc @fm3

Reply:
I think https://github.com/scalableminds/webknossos/blob/master/app/views/mail/validateUser.scala.html is what you are referring to.